.gitignore

```diff
@@ -36,3 +36,4 @@ benchmarks/param
 # pixi
 .pixi
 pixi.lock
+script.*
```

Comment:
```suggestion
```

I don't think personal files like this should be added to the `.gitignore`.

You can add it to a global `.gitignore` on your computer, this is what I do, an example of how to set it up [here](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
removed